modules/upgrade/pages/migrate/kubernetes/strimzi.adoc

```diff
@@ -0,0 +1,351 @@
+= Migrate from Strimzi to Redpanda Operator
+:description: This guide explores the migration from Strimzi to Redpanda Operator, highlighting key differences, deployment strategies, and how to leverage Kubernetes for managing Kafka clusters effectively.
+:env-kubernetes: true
+:page-categories: Migration
+:strimzi-version: 0.40.0
+:strimzi-link: https://github.com/strimzi/strimzi-kafka-operator/releases/tag/0.40.0
+
+In this guide, you'll explore the key differences between Strimzi and Redpanda Operator, and you'll find examples for migrating Strimzi custom resources to Redpanda.
+
+The information in this guide is based on version {strimzi-link}[{strimzi-version}] of the Strimzi Operator.
+
+== Why Redpanda?
+
+Redpanda is a Kafka-compatible event streaming platform that aims to simplify data streaming by eliminating external dependencies like ZooKeeper and offering a leaner architecture. See https://redpanda.com/blog/when-to-choose-redpanda-vs-kafka[When to choose Redpanda instead of Apache Kafka].
+
+== About Strimzi and Redpanda Operator in Kubernetes
+
+Both Strimzi and the Redpanda Operator harness the power of Kubernetes to provide tools for creating and managing Kafka clusters in Kubernetes environments:
+
+- Strimzi is an open-source project that simplifies the deployment and management of _Apache Kafka_ clusters on Kubernetes.
+
+- The Redpanda Operator simplifies the deployment and management of _Redpanda_ clusters on Kubernetes.
+
+The Redpanda Operator uses https://fluxcd.io/flux/concepts/[Flux^] to automate the deployment of the Redpanda Helm chart through the HelmRelease resource. For more details, see xref:deploy:deployment-option/self-hosted/kubernetes/k-deployment-overview.adoc[].
+
+image::shared:operator-flux-relationship.png[CRDs for the Redpanda Operator include Flux CRDs. Flux manages the Redpanda Helm chart through the HelmRelease resource.]
+
+== Notable differences
+
+These are some notable differences between features and functionality in Strimzi and Redpanda in Kubernetes.
+
+=== Security
+
+The Redpanda Helm chart emphasizes security out of the box, with the following defaults:
+
+- TLS enabled for the Kafka API, HTTP Proxy, and inter-broker communication (RPC).
+- Certificate management through glossterm:cert-manager[`cert-manager`] by default.
+
+NOTE: Both https://github.com/strimzi/strimzi-kafka-operator/issues/3088[Strimzi] and the Redpanda Operator currently do not support Kerberos for Kafka.
+
+See xref:manage:kubernetes/security/index.adoc[].
+
+=== Storage
+
+Redpanda performs best with direct-attached storage (NVMe or SSDs) to ensure high performance. To minimize latency, use of Kubernetes local PersistentVolumes.
+
+See xref:deploy:deployment-option/self-hosted/kubernetes/k-requirements.adoc[].
+
+=== JVM
+
+Redpanda does not require a JVM because it is written in C++.
```

Comment:
```suggestion
Redpanda does not require a JVM because it is written in C++ and does not require Zookeeper.
```

One of the reasons why it doesn't need a JVM is also because it doesn't require zookeeper. Not sure if we need to call this out tho

Reply:
Plus you already make the point about ZK below.